usaspending_api/disaster/helpers/covid_download_csv_strategies.py

```diff
@@ -0,0 +1,197 @@
+
+            delim = FILE_FORMATS[self.file_format]["delimiter"]
+
+            # Log how many rows we have
+            self._logger.info(f"Counting rows in delimited text file {temp_data_file_name}")
+            try:
+                count = count_rows_in_delimited_file(filename=temp_data_file_name, has_header=True, delimiter=delim)
+                self._logger.info(f"{destination_path} contains {count:,} rows of data")
+            except Exception:
+                self._logger.exception("Unable to obtain delimited text file line count")
+
+            start_time = time.perf_counter()
+            zip_process = multiprocessing.Process(
+                target=split_and_zip_data_files,
+                args=(
+                    str(covid_profile_download_zip_path),
+                    temp_data_file_name,
+                    str(destination_path),
+                    self.file_format,
+                    None,
+                ),
+            )
+            zip_process.start()
+            wait_for_process(zip_process, start_time, None)
+        except Exception as e:
+            raise e
+        finally:
+            Path(temp_file_path).unlink()
+        return [destination_path], count
+
+
+class DatabricksToCSVStrategy(AbstractToCSVStrategy):
+    def __init__(self, logger: logging.Logger, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._logger = logger
+
+    def download_to_csv(
+        self, source_sql, destination_path, destination_file_name, working_dir_path, covid_profile_download_zip_path
+    ):
+        self.spark = None
+        # The place to write intermediate data files to in s3
+        s3_bucket_name = settings.BULK_DOWNLOAD_S3_BUCKET_NAME
+        s3_bucket_path = f"s3a://{s3_bucket_name}"
+        s3_destination_path = f"{s3_bucket_path}/temp_covid_download/{destination_file_name}"
+        try:
+            extra_conf = {
+                # Config for Delta Lake tables and SQL. Need these to keep Dela table metadata in the metastore
+                "spark.sql.extensions": "io.delta.sql.DeltaSparkSessionExtension",
+                "spark.sql.catalog.spark_catalog": "org.apache.spark.sql.delta.catalog.DeltaCatalog",
+                # See comment below about old date and time values cannot be parsed without these
+                "spark.sql.legacy.parquet.datetimeRebaseModeInWrite": "LEGACY",  # for dates at/before 1900
+                "spark.sql.legacy.parquet.int96RebaseModeInWrite": "LEGACY",  # for timestamps at/before 1900
+                "spark.sql.jsonGenerator.ignoreNullFields": "false",  # keep nulls in our json
+            }
+            self.spark = get_active_spark_session()
+            self.spark_created_by_command = False
+            if not self.spark:
+                self.spark_created_by_command = True
+                self.spark = configure_spark_session(**extra_conf, spark_context=self.spark)  # type: SparkSession
```

Comment:
I wonder if there is a way that we can reuse the spark session across files. Not a dealbreaker for this PR. Just something to consider in the future. Might want to see how this would affect performance.